This patch has been taken from ArchLinux. It should fix the yodl build
on riscv64 and it's generally very undesirable to use tput(1) for
buildlogs generated by the Alpine builders.

commit 42d3987d08e41723a2ca8739550bbf5d58e45ea8
Author: Baptiste Jonglez <git@bitsofnetworks.org>
Date:   Sat Jul 7 12:58:54 2018 +0200

    Disable screen clearing
    
    This tends to fail when building in a clean chroot (e.g. yodl) because of
    missing terminal definition, and is of dubious interest anyway.

diff --git a/icmake/scripts/icmbuild.in b/icmake/scripts/icmbuild.in
index c31e925..dba9607 100644
--- a/icmake/scripts/icmbuild.in
+++ b/icmake/scripts/icmbuild.in
@@ -618,7 +618,7 @@ void install(string what, string path)
 
 void clearScreen(int cls)
 {
-    if (cls)
+    if (0)
         system("tput clear");
 }
 
